notebooks/03-query/15-pivot-unpivot.ipynb

```diff
@@ -0,0 +1,433 @@
+    "display_client = druid.display\n",
+    "sql_client = druid.sql\n",
+    "status_client = druid.status\n",
+    "\n",
+    "status_client.version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "645f0209-436a-47e2-a1f4-be6c43de0eac",
+   "metadata": {},
+   "source": [
+    "## Load example data\n",
+    "\n",
+    "Once your Druid environment is up and running, ingest the sample data for this tutorial.\n",
+    "\n",
+    "Run the following cell to create a table called \"example-wiki-pivot-unpivot\". When completed, you'll see a description of the final table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "032ee776-1eb9-46dc-989a-933bd7c7d1da",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "REPLACE INTO \"example-wiki-pivot-unpivot\" OVERWRITE ALL\n",
+    "WITH \"ext\" AS (\n",
+    "  SELECT *\n",
+    "  FROM TABLE(\n",
+    "    EXTERN(\n",
+    "      '{\"type\":\"http\",\"uris\":[\"https://druid.apache.org/data/wikipedia.json.gz\"]}',\n",
+    "      '{\"type\":\"json\"}'\n",
+    "    )\n",
+    "  ) EXTEND (\"isRobot\" VARCHAR, \"channel\" VARCHAR, \"timestamp\" VARCHAR, \"flags\" VARCHAR, \"isUnpatrolled\" VARCHAR, \"page\" VARCHAR, \"diffUrl\" VARCHAR, \"added\" BIGINT, \"comment\" VARCHAR, \"commentLength\" BIGINT, \"isNew\" VARCHAR, \"isMinor\" VARCHAR, \"delta\" BIGINT, \"isAnonymous\" VARCHAR, \"user\" VARCHAR, \"deltaBucket\" BIGINT, \"deleted\" BIGINT, \"namespace\" VARCHAR, \"cityName\" VARCHAR, \"countryName\" VARCHAR, \"regionIsoCode\" VARCHAR, \"metroCode\" BIGINT, \"countryIsoCode\" VARCHAR, \"regionName\" VARCHAR)\n",
+    ")\n",
+    "SELECT\n",
+    "  TIME_PARSE(\"timestamp\") AS \"__time\",\n",
+    "  \"isRobot\",\n",
+    "  \"channel\",\n",
+    "  \"added\",\n",
+    "  \"user\",\n",
+    "  \"deleted\"\n",
+    "FROM \"ext\"\n",
+    "PARTITIONED BY DAY\n",
+    "'''\n",
+    "display_client.run_task(sql)\n",
+    "sql_client.wait_until_ready('example-wiki-pivot-unpivot')\n",
+    "display_client.table('example-wiki-pivot-unpivot')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36ad757a-957e-465a-a284-855c14e3e8fd",
+   "metadata": {},
+   "source": [
+    "## PIVOT\n",
+    "\n",
+    "PIVOT is a SQL operator that is used after the FROM clause that reduces rows in a resultset by converting rows into columns.\n",
+    "```\n",
+    "    FROM <data source>\n",
+    "    \n",
+    "    PIVOT ( <list of aggregations>\n",
+    "            \n",
+    "            FOR <pivot_column> IN (<list of values in pivot_column>)\n",
+    "          )\n",
+    "```\n",
+    "- \\<list of aggregations\\> - list of aggregate expressions, example : SUM(added) as added, SUM(deleted) as deleted \n",
+    "- \\<pivot_column\\> - the column whose values are being turned into new columns\n",
+    "- \\<list of values in pivot_column\\> to use in the form `<literal value> as <value_name>,...`, Example: 'true' as robot, 'false as human \n",
```

Comment:
```suggestion
    "- \\<list of values in pivot_column\\> - to use in the form `<literal value> as <value_name>,...`; for example: 'true' as robot, 'false as human \n",
```
Consider adding a short description. What is this list for?

Reply:
The explanatino is the next paragraph, I found it kinda difficult to describe in the parameter list because this and the list of aggregates are combined to produce the resulting columns. This is the sentence:
"The operation produces columns in the form \<value_name\>__\<aggregation_name\> for all the aggregations and values that are specified."